### PR TITLE
Fix tsconfig's "lib" property, along with small improvements to README.md

### PR DESCRIPTION
**Pull request by BloodShura**
This pull request:

1. fixes incorrect value provided to `tsconfig.json`'s `lib` property (`DOM` -> `dom`)
2. fixes small typos on `README.md`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -213,7 +213,7 @@ function App () {
 
 _If you want to pass variables to a GraphQL query, check out [Multiple Arguments](#multiple-arguments)._
 
-Note that `fetcher` can be skipped from the parameters if it's provided gloablly.
+Note that `fetcher` can be omitted from the parameters if it's provided gloablly.
```
Comment (pacocoursey) [inline README.md]:
```suggestion
Note that `fetcher` can be omitted from the parameters if it's provided globally.
```

**Comment by pacocoursey** [comment]
Everything else looks good, thank you!

File: README.md [hunk 2]
```diff
@@ -21,7 +21,7 @@
 
 SWR is a React Hooks library for remote data fetching.
 
-The name “**SWR**” is derived from `stale-while-revalidate`, a cache invalidation strategy popularized by [HTTP RFC 5861](https://tools.ietf.org/html/rfc5861).
```
Comment (quietshu) [inline README.md]: This is intended (using 2 spaces to start a new line).

File: README.md [hunk 3]
```diff
@@ -266,7 +266,7 @@ useSWR(['/api/data', token], fetchWithToken)
 ```
 
 This solves the problem. The key of the request is now the combination of both values. SWR **shallowly** compares
-the arguments on every render, and triggers revalidation if any of them has changed.
```
Comment (quietshu) [inline README.md]: same here

File: README.md [hunk 4]
```diff
@@ -341,7 +341,7 @@ function Profile () {
 Clicking the button in the example above will send a POST request to modify the remote data, locally update the client data and
 try to fetch the latest one (revalidate).
 
-But many POST APIs will just return the updated data directly, so we don’t need to revalidate again.
```
Comment (quietshu) [inline README.md]: and here

File: README.md [hunk 5]
```diff
@@ -174,7 +174,7 @@ function App () {
 
 ### Data Fetching
 
-`fetcher` is a function **accepts the `key`** of SWR, and returns a value or a Promise.
```
Comment (quietshu) [inline README.md]: This should end up with a new line as well.

**Comment by BloodShura** [comment]
Oops, sorry about that. Automatic whitespace trimming was enabled in my editor. Should be good now.

File: README.md [hunk 6]
```diff
@@ -213,7 +213,7 @@ function App () {
 
 _If you want to pass variables to a GraphQL query, check out [Multiple Arguments](#multiple-arguments)._
 
-Note that `fetcher` can be skipped from the parameters if it's provided gloablly.
+Note that `fetcher` can be ommited from the parameters if it's provided globally.
```
Comment (pacocoursey) [inline README.md]:
```suggestion
Note that `fetcher` can be omitted from the parameters if it's provided globally.
```

**Pull request merged**